src/lib/helpers.ts

```diff
@@ -249,6 +249,8 @@ export async function generatePKCEVerifier() {
     for (let i = 0; i < verifierLength; i++) {
       array[i] = Math.floor(Math.random() * 256)
     }
+  } else {
+    window.crypto.getRandomValues(array)
```

Comment:
Do we need to make `array` `var` and assign it here?
```suggestion
    array = window.crypto.getRandomValues(array)
```

Reply:
Ah, sorry, never mind. Just verified that the current code works!